app/services/gias/csv_transformer.rb

```diff
@@ -56,7 +56,7 @@ def school_in_scope?(row)
     end
 
     class School
-      OPEN_SCHOOL = "1".freeze
+      OPEN_SCHOOL_CODES = %w[1 3].freeze # Open and proposed to be closed but still open
```

Comment:
Can you please annotate what the numeric codes mean? For example, see `NON_ENGLISH_ESTABLISHMENTS` below.

https://github.com/DFE-Digital/itt-mentor-services/blob/4392230bbb5de69a64eca6483fc53876223c5c0f/app/services/gias/csv_transformer.rb#L60-L64

This'll help us understand what the codes mean in future, when we've all forgotten what `1` and `3` mean 🙈